Iceberg-TipUI/IceTipMergeCommand.class.st

```diff
@@ -10,13 +10,13 @@ Class {
 { #category : #'accessing - defaults' }
 IceTipMergeCommand class >> defaultDescription [
 
-	^ 'Merge your current branch with another.'
+	^ 'Merge a given branch into your current branch.'
 ]
 
 { #category : #'accessing - defaults' }
 IceTipMergeCommand class >> defaultName [
 
-	^ 'Merge'
+	^ 'Merge into current'
```

Comment:
and is this thing what is making tests fail. 
also, merges are always "into current", I do not see the reason to put this explanation :)